pkg/util/test/util.go

```diff
@@ -277,32 +280,118 @@ func ValidateStorageCluster(
 		return err
 	}
 
+	k8sVersion, err := coreops.Instance().GetVersion()
+	if err != nil {
+		return err
+	}
+
+	if isPVCControllerEnabled(cluster) {
```

Comment:
nit: Can we extract all the component validations in a single separate function for readability?